frame/nomination-pools/src/migration.rs

```diff
@@ -52,9 +52,12 @@ pub mod v1 {
 
 	impl<T: Config> OldBondedPoolInner<T> {
 		fn migrate_to_v1(self) -> BondedPoolInner<T> {
+			// Note: `commission` field not introduced to `BondedPoolInner` until
+			// migration 4.
```

Comment:
would be good if you link to https://github.com/paritytech/substrate/issues/13024

Reply:
Seeing this, I am more slightly inclined to just close that issue before this PR is merged :))